client/scss/components/_userbar.scss

```diff
@@ -216,20 +220,14 @@ $positions: (
 
 .w-dialog--userbar {
   // Display off to the side of the page rather than in the middle.
-  inset-inline-start: auto;
   font-family: theme('fontFamily.sans');
-  padding-inline-end: 2rem;
   z-index: $userbar-z-index;
 
   .w-dialog__close-button {
-    $size: theme('spacing.6');
-    width: $size;
-    height: $size;
-    top: calc(-1 * $size / 2);
-    inset-inline-end: calc(-1 * $size / 2);
     border-radius: theme('borderRadius.full');
     border: 2px solid theme('colors.icon-primary');
     background: theme('colors.surface-page');
+    z-index: calc($userbar-z-index + 1);
```

Comment:
Without this, the close button is partly obscured by the dialog content when the position is on the left